src/client/content_mapblock.cpp

```diff
@@ -67,6 +67,17 @@ MapblockMeshGenerator::MapblockMeshGenerator(MeshMakeData *input, MeshCollector
 	nodedef   = data->m_client->ndef();
 	meshmanip = RenderingEngine::get_scene_manager()->getMeshManipulator();
 
+	std::string clean_nodebox_textures = g_settings->get("clean_nodebox_textures");
```

Comment:
const std::string &

Reply:
Done.
By the way, just noticed that `Settings` is not entirely thread-safe: concurrent access and change of the same entry is unsafe.